talent/job-descriptions/devrel-sf.md

```diff
@@ -0,0 +1,26 @@
+Developer Advocate / Tech Writer
+------------------------------------------
+
+### [Location & Seniority](../location-seniority-section.md)
+
+**Compensation:** ? [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md) for ?
```

Comment:
The salary should be discussed within management, but if it's a salary adjusted to the market and every candidate's expectations, we could add something like "Compensation: competitive market salary" as we did for the ```Applications engineer in SF```.

Reply:
SGTM, what do you think, @eiso ?